app/AppCache.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+require_once __DIR__.'/AppKernel.php';
+
+use Symfony\Bundle\FrameworkBundle\HttpCache\HttpCache;
+
+class AppCache extends HttpCache
+{
+
+
+
+var $hogeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee='fugaaaaaa';
+
+
+
+}
```

Comment:
__[development][code_sniffer]__
* The closing brace for the class must go on the next line after the body